### PR TITLE
add 383. Ransom Note

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m
I'm sorry for the late submission. Could you check my code ?

### CODE REVIEW

File: 383_ransom-note.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# @lc code=start
+class Solution:
+    def canConstruct(self, ransomNote: str, magazine: str) -> bool:
+        """
+        time: O(n)
+        space: O(n)
+        n : max(len(ransomNote), len(magazine))
+        """
+        ransomNote_dict = collections.Counter(ransomNote)
```
Comment (ryuji0123) [inline 383_ransom-note.py:17]: ransomNote_counter

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I corrected unsuitable names.  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

File: 383_ransom-note.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#
+# @lc app=leetcode id=383 lang=python3
+#
+# [383] Ransom Note
+#
+import collections
```
Comment (tkkawa) [inline 383_ransom-note.py:6]: from collections import Counter

**Review by tkkawa — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I modified my code. Please check it.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**